From e7c853ab35cbef71e8c0d82c486fd7fa3d7c34da Mon Sep 17 00:00:00 2001
From: Adithya R <gh0strider.2k18.reborn@gmail.com>
Date: Sat, 10 Apr 2021 20:07:15 +0530
Subject: [PATCH 5/8] inputflinger: Enable ThinLTO

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
---
 services/inputflinger/Android.bp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/services/inputflinger/Android.bp b/services/inputflinger/Android.bp
index b332c42..4c3a264 100644
--- a/services/inputflinger/Android.bp
+++ b/services/inputflinger/Android.bp
@@ -55,6 +55,9 @@ cc_defaults {
         "libutils",
         "libui",
     ],
+    lto: {
+        thin: true,
+    },
 }
 
 cc_library_shared {
-- 
2.31.1

